### PR TITLE
national insurance error

### PR DESCRIPTION
**Pull request by dbjsdev**
https://trello.com/c/6yZLnek8/31-staff-records-errors-and-warnings-national-insurance-number

This is an initial PR to ensure it is as intended.

### CODE REVIEW

File: server/models/BulkImport/csv/workers.js [hunk 1]
```diff
@@ -2427,6 +2427,20 @@ class Worker {
     };
   }
 
+  // Exceeds national insurance maximum
+  exceedsNationalInsuranceMaximum(originalLineNumber) {
+    return {
+      origin: 'Workers',
+      lineNumber: this._lineNumber,
+      errCode: Worker.DUPLICATE_ERROR,
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: You need to create a new constant for this new error code.

File: server/models/BulkImport/csv/workers.js [hunk 2]
```diff
@@ -2427,6 +2427,20 @@ class Worker {
...
+      errType: `DUPLICATE_ERROR`,
+      error: `NINUMBER is already associated with another full time worker record`,
+      source: this._currentLine.UNIQUEWORKERID,
+      worker: this._currentLine.UNIQUEWORKERID,
+      name: this._currentLine.LOCALESTID,
```
Comment (wozzer72) [inline server/models/BulkImport/csv/workers.js]: this should be the `NINUMBER` field, not `LOCALESTID`.

File: server/models/BulkImport/csv/workers.js (hunk 1 above)
Comment (dbjsdev) [reply server/models/BulkImport/csv/workers.js]: Ah, I was unsure if it had a different error code, would NI_HOURS_ERROR be acceptable?

File: server/models/BulkImport/csv/workers.js (hunk 2 above)
Comment (dbjsdev) [reply server/models/BulkImport/csv/workers.js]: change applied

**Pull request merged**